deploy-helm/helm-operator-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: flux-helm-operator
+  # helm-operator should be created in the same namespace
+  # as flux agent
```

Comment:
Why is that?

Reply:
I initially thought it was needed to be to get at the secret, but thinking about it now, it's not. Fixing.

Reply:
If it does need access to the same git deploy key secret, yes it will need to be in the same namespace. If so, it's worth explaining why in the comment.